### DISCUSSION TITLE
Tesla Android version 2023.x with a Pi400?

### DISCUSSION THREAD

**Discussion by DORNINEM** [tesla-android/issue-tracker#216] [Q&A]
Hi,
I tried to setup the lasted version on my raspberry pi400 and... After 2 automatic boot up the screen (I connected to a external display) remains blank and nothing happens 😭
Does anyone use a pi400 with success or is it a bug?

**Comment by GtHbbrr** [comment]
https://twitter.com/mikegapinski/status/1622502391621775360?s=20&t=wKL1lzXPQdwhKMd5V3DdWg
